static/js/composebox_typeahead.js

```diff
@@ -332,18 +354,24 @@ exports.content_typeahead_selected = function (item) {
         beginning = (beginning.substring(0, beginning.length - this.token.length-1)
                 + '#**' + item.name + '** ');
         $(document).trigger('streamname_completed.zulip', {stream: item});
+    } else if (this.completing === 'syntax') {
+        beginning = "```" + item + "\n";
```

Comment:
note to self: Fix this today, this has a side effect of deleting anything the user has already typed before/after the backticks